samples/sampler/stories/qa/ColorPicker.js

```diff
@@ -0,0 +1,46 @@
+import ColorPicker, {ColorPickerBase} from 'ColorPicker';
```

Comment:
```suggestion
import ColorPicker, {ColorPickerBase} from '@enact/sandstone/ColorPicker';
```
QA Sampler does not run without this change.

Reply:
@0x64 I pushed the changes, could you review them again, please?

Reply:
@ion-andrusciac-lgp I checked updated changes. Thank you!